modules/GenericPowermeter/serialization.hpp

```diff
@@ -0,0 +1,178 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Qwello GmbH and Contributors to EVerest
+
+/// @brief Serialization and deserialization for the modbus interface.
+///
+/// The format has a major quirk: Even though our exchanged data is four bytes
```

Comment:
can you add a static_assert for this somehow?

Reply:
Would be nice but I don't think so.. its not expressed through the type system